core/storage/user/gae_models.py

```diff
@@ -67,6 +67,20 @@ def get_by_normalized_username(cls, normalized_username):
             cls.normalized_username == normalized_username).get()
 
 
+class UserContributionsModel(base_models.BaseModel):
+    """Tracks statistics for a particular user.
+    Includes subsequently deleted and private explorations.
```

Comment:
I think this is more appropriate as a description for each attribute -- I'll make suggestions below.